source/includes/_tipser-elements-analytics.md

```diff
@@ -74,6 +74,34 @@ The code above assumes that you use <a href="https://developers.google.com/analy
 
 For the instructions how to setup analytics.js script on your site and connect it to your GA account, refer to the <a href="https://developers.google.com/analytics/devguides/collection/analyticsjs" target="_blank">official documentation</a>.
 
+## Typical use case: Facebook Pixel
+
+Facebook Pixel is also an option to tunnel Tipser Analytics events, here's a snippet:
+
+```javascript
+document.addEventListener('tw-track', function(e) {
+    // fbq() function coming from Facebook Pixel analytics script
+    fbq('trackCustom', e.detail.action, {
+        content_name: e.detail.target, 
+        content_category: e.detail.description,
+        content_type: e.detail.action,
+    });
+});
+```
+Example above doesn't translate tipser events to pixel events, you will have to make corelation in pixel dashboard or you can use function that translates those for you. 
+
+1. Go <a href="https://gist.github.com/sirpeas/114bba643e920b0f53dff9487670f509" target="_blank">here</a> and copy the script.
```

Comment:
I wonder if it's possible to publish a gist as an organization (Tipser) rather than as a specific user. It would look much more professional :). But if it's not possible then let it be 🤷‍♀ .

Reply:
we can always put this into some our misc repo, and link from there instead of gist

Reply:
Or we can create snippet list in docs, which one is better?

Reply:
Would the snippet be "collapsible"? Or would occupy the full height? I think that the section for FB Pixel integration (which is still relatively niche topic) shouldn't occupy vast spaces of our docs.

Reply:
Can be collapsible 
https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab

Reply:
That would work!